app/src/main/java/com/example/androidtrainingproject/MainActivity.kt

```diff
@@ -37,6 +92,130 @@ fun Greeting(name: String, modifier: Modifier = Modifier) {
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun LoginScreen(navigateToProductDetails: () -> Unit = {}) {
+//    var (textField, setTextField) = remember { mutableStateOf("") }
+    val usernameField = remember { mutableStateOf("") }
+    val passwordField = remember { mutableStateOf("") }
+
+    val isPasswordVisible = remember { mutableStateOf(false) }
+
+    val hardcodedUsername = "admin@test.com"
+    val hardcodedPassword = "1234"
+
+    Column(modifier = Modifier.padding(10.dp)) {
+        Text(text = "Log in", fontSize = 26.sp , modifier = Modifier
+            .padding(bottom = 10.dp)
+
+        )
+        TextField(
+            value = usernameField.value,
+            onValueChange = {
+                newValue -> usernameField.value = newValue
+            },
+            modifier = Modifier
```

Comment:
Note: Just an advice - always place `modifier` as the first argument. This way you have a better overall visibility of how the composable is positioned at just a glance.

Reply:
ok, sure, I've moved all modifiers on the first lines